android_sample/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Fri Sep 28 15:42:15 PDT 2018
```

Comment:
Can't add a comment on it, but what is the jar file for? Do we have distribution rights for that jar file?